homework_1/task_01/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+"flag"
+"fmt"
+"log"
+"strings"
+
+"code-cadets-2021/homework_1/task_01/fizzBuzz"
+)
```

Comment:
Missing tabs, bad formatting

Reply:
I don't know whats up with that, I've seen it more files after I commited and pushed. There are indents in Goland and VSCode both.